### PR TITLE
Adds Cloud Storage sample for AppEngine Flex

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: appengine/flexible/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Cloud Storage & Google App Engine
+
+This sample application demonstrates how to use [Cloud Storage with Google App Engine](https://cloud.google.com/appengine/docs/flexible/php/using-cloud-storage).
+
+## Setup
+
+Before running this sample:
```
Comment (tmatsuo) [inline appengine/flexible/storage/README.md]: Probably worth adding project and bucket creation?

File: appengine/flexible/storage/app.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    <h1>Storage Example</h1>
+    <h3>Write [<a href="https://cloud.google.com/appengine/docs/flexible/php/using-cloud-storage">docs</a>]:</h3>
+    <form action="/write" method="post">
+        Some file content:<br />
+        <textarea name="content"></textarea><br />
+        <input type="submit" />
+    </form>
+EOF;
+    if ($content) {
+        $form .= "<p><strong>Your content:</strong><p><p>$content</p>";
```
Comment (tmatsuo) [inline appengine/flexible/storage/app.php]: The `$content` is not sanitized, so it's basically XSS vulnerable.

I understand the rationale of using inline html for simplicity, but I would use html template library and let it sanitize the value.

File: appengine/flexible/storage/app.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+runtime: php
+vm: true
+
+# [START env_variables]
+env_variables:
+  GOOGLE_BUCKET_NAME: "your-bucket-name"
+# [END env_variables]
```
Comment (tmatsuo) [inline appengine/flexible/storage/app.yaml]: Small nit: add newline?

File: appengine/flexible/storage/app.php (hunk 2 above)
Comment (bshaffer) [inline appengine/flexible/storage/app.php]: How about we use `htmlspecialchars` instead?

Comment (tmatsuo) [inline appengine/flexible/storage/app.php]: Sounds ok too

File: appengine/flexible/storage/README.md (hunk 1 above)
Comment (bshaffer) [inline appengine/flexible/storage/README.md]: added!

File: appengine/flexible/storage/app.yaml (hunk 3 above)
Comment (bshaffer) [inline appengine/flexible/storage/app.yaml]: done

**Pull request merged**